data/data-pipeline/data_pipeline/comparison_tool/src/run_tract_comparison.py

```diff
@@ -0,0 +1,82 @@
+"""
+This script is a 'starter' for parameterized work pertaining to generating comparisons.
+Here, we only compare  DAC lists that operate at the tract level. We can change/expand this as we move forward.
+
+Why papermill? Papermill is an easy way to parameterize notebooks. While doing comparison work,
+I often had to do a lot of one-off analysis in a notebook that would then get discarded. With parametrized notebooks,
+we can run each type of analysis and then store it for posterity. The first template is just for agencies, but you could
+imagine generating interactive outputs for many types of analysis, like demographic work.
+
+To see more: https://buildmedia.readthedocs.org/media/pdf/papermill/latest/papermill.pdf
+
+To run:
+` $ python src/run_tract_comparison.py --template_notebook=TEMPLATE.ipynb --parameter_yaml=PARAMETERS.yaml`
+"""
+
+import os
+import datetime
+import argparse
+import yaml
+import papermill as pm
+
+
+def _read_param_file(param_file: str) -> dict:
+    """Reads params and enforces a few constraints:
+
+    1. There's a defined output path
+    2. Change all relative paths to absolute paths
+    """
+    with open(param_file, "r", encoding="utf8") as param_stream:
+        param_dict = yaml.safe_load(param_stream)
+    assert (
+        "OUTPUT_DATA_PATH" in param_dict
```

Comment:
This could all get replaced by Marshmallow, for instance.

Reply:
added an issue